ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# Orbit Web
+
+The current implementation status, immediate-term plans, and future goals of this repository.
+
+We welcome feature requests as edits to the "[Backlog](#backlog)" section below. Suggest something by opening a pull request editing this file.
+
+## Build Status
+
+TBA
+
```

Comment:
We can prolly remove the "Build Status" section? It seems to me it's not required for the purposes of this roadmap.